src/Cardano/Wallet/Network/HttpBridge.hs

```diff
@@ -150,14 +156,25 @@ mkHttpBridge mgr baseUrl network = HttpBridge
         run (map getApiT <$> getEpochById network (EpochIndex ep))
     , getNetworkTip =
         run (blockHeaderHash <$> getTipBlockHeader network)
+    , postSignedTx =
+        -- TODO: We would be good to parse errors from the node
+        -- like "Failed to send to peers: Blockchain protocol error"
+        -- and "Transaction failed verification: transaction has no inputs"
+        toUnit . run . (_postSignedTx network) . ApiT
     }
   where
     run :: ClientM a -> ExceptT HttpBridgeError IO a
     run query = ExceptT $ (first convertError) <$> runClientM query env
+
     env = mkClientEnv mgr baseUrl
+
+    toUnit :: Monad m => ExceptT e m NoContent -> ExceptT e m ()
+    toUnit = mapExceptT (>>= return . void)
```

Comment:
And what about just: `void` :sweat_smile:  ?

```hs
-- For every 'Monad'
void : Monad m => m a -> m ()

-- 'ExceptT e m' is actually a 'Monad', provided that 'm' is one
instance Monad m => Monad (Except e m)
```

Reply:
Ha! 😄 Fixed.